boards/BMS/Src/App/App_LatchStatus.c

```diff
@@ -0,0 +1,28 @@
+#include <stdlib.h>
+#include <assert.h>
+#include "App_LatchStatus.h"
+
+struct LatchStatus
+{
+    bool (*is_enabled)(void);
+};
+
+struct LatchStatus *App_LatchStatus_Create(bool (*is_enabled)(void))
+{
+    struct LatchStatus *latch_status = malloc(sizeof(struct LatchStatus));
+    assert(latch_status != NULL);
+
+    latch_status->is_enabled = is_enabled;
+
+    return latch_status;
+}
+
+void App_LatchStatus_Destroy(struct LatchStatus *latch_status)
+{
+    free(latch_status);
+}
+
+bool App_LatchStatus_IsEnabled(const struct LatchStatus *latch_status)
+{
+    return latch_status->is_enabled();
```

Comment:
Hmm, do we just have a requirement that `is_enabled()` will latch?

Reply:
The specs were confusing. I'm chatting with @kirons96 about this and it should be renamed to OkStatus probably. Will get back to this.